packages/app/src/components/authorize/AuthorizeToken.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, MouseEventHandler } from "react";
+import { Alert, Tooltip, OverlayTrigger } from "react-bootstrap";
+import { constants } from "ethers";
+import { addresses } from "@tender/contracts";
+import { BigNumber } from "@ethersproject/bignumber";
+
+type Props = {
+  protocolName: string;
+  tokenSymbol: string;
+  isTokenAuthorized: boolean;
```

Comment:
this is incorrect terminology

Reply:
what are you suggesting? In my understanding approval is for a specific amount, and we are trying to use it for a general "authorization", that was my thinking behind the naming.